apps/web/pages/event-types/index.tsx

```diff
@@ -770,14 +771,12 @@ const CreateFirstEventTypeView = () => {
   );
 };
 
-const CTA = () => {
+const CTA = ({ data }: { data: GetByViewerResponse }) => {
   const { t } = useLocale();
 
-  const query = trpc.viewer.eventTypes.getByViewer.useQuery();
```

Comment:
we were using this twice with different inputs

trpc.viewer.eventTypes.getByViewer.useQuery();

trpc.viewer.eventTypes.getByViewer.useQuery(filters && { filters });